test/specs/repo/extensions/wikibase-local-media.ts

```diff
@@ -76,10 +77,6 @@ describe( 'WikibaseLocalMedia', function () {
 		await propertyIdSelector( propertyId ).click();
 
 		await browser.keys( 'image.png'.split( '' ) );
-		await $( 'ul.ui-mediasuggester-list' )
-			.$( 'a' )
-			.$( 'span[title="Image.png"]' )
-			.click();
```

Comment:
should this really be changed as part of this PR?

Reply:
It had to be to make the tests pass. Due to additional latencies with the Job Runner based config it wouldn't pass reliably as it was written.

Reply:
Also -- to be clear, this change and the test change above, were both flakey at the time of merging from the "tests run against Example" PR, they were just further fixed here.

Reply:
Got it